schema/dex/trades.sql

```diff
@@ -446,7 +446,26 @@ WITH rows AS (
             tx_hash,
             NULL::integer[] AS trace_address,
             evt_index_trades
-        FROM gnosis_protocol."view_trades"
+        FROM gnosis_protocol.view_trades
+
+        UNION
+
+        -- Bancor Network
+        SELECT
+            block_time,
+            'Bancor Network' AS project,
+            NULL AS version,
+            trader AS trader_a,
+            NULL::bytea AS trader_b,
+            target_token_amount_raw AS token_a_amount_raw,
+            source_token_amount_raw AS token_b_amount_raw,
+            target_token_address AS token_a_address,
+            source_token_address AS token_b_address,
```

Comment:
Just realized that these might be wrong: should not `token_a` be the `source` and `token_b` be the `target`?

Reply:
@hagaetc wdyt?